gateway_tests/prod_config/conftest.py

```diff
@@ -0,0 +1,152 @@
+# Goals:
```

Comment:
I like the functionality here, but I think it may belong in some "util.py" like the other test suite. 
I'm planning on moving up `conftest.py` in the other PR to include generally useful "start gateway, start IOC" stuff that will be shared.

Reply:
A totally reasonable approach, for sure!

For "mega util.py or not" - I guess if it grows large enough, we could break it up into `gateway_tests.util.xyz`. Given our timeline, I'd guess it won't grow so much to require that. How about for now:
1. If it seems like it could be shared among everything, throw it in `gateway_tests.util` (or conftest?)
2. If it seems sub-test specific, throw it in `gateway_tests.subtest.util`

If (2) ends up getting reused, move specific utils up to (1)

Reply:
Sounds good to me. I'll move this up before this part of the suite is done.